Week2/js-exercises/collectiveAge.js

```diff
@@ -0,0 +1,31 @@
+const hackYourFutureMembers = [
+    { name: 'Wouter', age: 33 },
+    { name: 'Federico', age: 32 },
+    { name: 'Noer', age: 27 },
+    { name: 'Tjebbe', age: 22 },
+];
+
+function combinedAgesArr(arrayOfObjects) {
+
+    let total = 0;
+
+    //for this stage i used a callback function with an arrow form
+
+    let agesArr = arrayOfObjects.map((object) => {
+        return object["age"]
+    })
```

Comment:
Very good! I would just suggest naming the variable `object` to something more descriptive. Like `person`.
Also you can use `person.age` notation which is more readable than `person["age"]`